services/web-app/utils/use-query-state.js

```diff
@@ -25,75 +22,48 @@ const queryStringConfig = {
  * 1- the value contained in `value` will automatically be updated when the browser query changes
  * 2- subscribing components can manipulate the query value by calling the `update`
  * function with a desired value
- * 3 - All queryState key/values will be saved to localStorage
- * 4 - The `validateValue` function receives the current query string value
+ * 3 - The `validateValue` function receives the current query string value
  * and should return true if the value is valid or false otherwise. If the value is invalid,
- * the hook will return the last value saved to localStorage (null if not set)
- * 5 - By default, queryStates are always saved to localStorage and this value is reset
- * to the provided `defaultValue` every time a new page navigation to the path occurs;
- * supplying option `resetOnLoad` = `false` will cause the value to never be reset
- * 6- Supplying options `parseNumbers` or `parseBoolean` = `true` will cause the type of value
+ * the hook will return the defaultValue ('' if not set)
+ * 4- Supplying options `parseNumbers` or `parseBoolean` = `true` will cause the type of value
  * to be cast to desired type if possible, otherwise type will be string
  * @param {string} key Key to use in the query string
  * @param {{defaultValue: string, parseNumbers: boolean,
- * parseBooleans: boolean, resetOnLoad: boolean}} options Extra config options
+ * parseBooleans: boolean}} options Extra config options
```

Comment:
Same comments on this function as above about return type consistency and param naming.

Reply:
Done!